aspnetcore/blazor/hybrid/security/maui-blazor-web-identity.md

```diff
@@ -0,0 +1,226 @@
+---
+title: .NET MAUI Blazor Hybrid with a Blazor Web App and ASP.NET Core Identity
+author: guardrex
+description: Learn about the .NET MAUI Blazor Hybrid with a Blazor Web App that manages authentication with ASP.NET Core Identity sample app.
+monikerRange: '>= aspnetcore-9.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 03/12/2025
+uid: blazor/hybrid/security/maui-blazor-web-identity
+---
+# .NET MAUI Blazor Hybrid with a Blazor Web App and ASP.NET Core Identity
+
+<!-- UPDATE 10.0 - Uncomment at 10.0
+
+[!INCLUDE[](~/includes/not-latest-version.md)]
+
+-->
+
+The .NET MAUI Blazor Hybrid with a Blazor Web App and ASP.NET Core Identity sample app demonstrates how to share a common UI and with ASP.NET Core Identity local accounts. Although ASP.NET Core Identity is used, you can use this pattern for any authentication provider from a MAUI Blazor Hybrid client.
```

Comment:
```suggestion
This sample app demonstrates how to build a .NET MAUI Blazor Hybrid app with a Blazor web app that uses ASP.NET Core Identity. It shows how to share common UI and authenticate with ASP.NET Core Identity local accounts. Although ASP.NET Core Identity is used, you can use this pattern for any authentication provider from a MAUI Blazor Hybrid client.
```

Reply:
@BethMassi ... This one needs a closer look because that type of a Blazor app is a proper noun. Can it be ...

```diff
- Blazor web app
+ Blazor Web App
```

... to match all of the references to BWAs across the docs?

Reply:
*BTW* ... I updated the sample repo's README with the changes that we just made here on ...

https://github.com/dotnet/blazor-samples/pull/487

Reply:
Sure.